src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -681,20 +682,21 @@ describe('ReactUpdates', () => {
       }
 
       componentDidMount() {
+        this.container = document.createElement('div');
```

Comment:
It does work the old way but it warns. Stack doesn't warn but it seems like it should?

Why doesn't Stack warn?